cli/.env-default

```diff
@@ -0,0 +1,4 @@
+ELATION_BASE_URL=https://app.elationemr.com/api/2.0/
```

Comment:
Can we have the default vars point to the sandbox env?

Reply:
Sure, can do.